gpiozero/boards.py

```diff
@@ -265,7 +265,7 @@ class LEDBarGraph(LEDCollection):
     """
 
     def __init__(self, *pins, **kwargs):
-        super(LEDBarGraph, self).__init__(*pins, pwm=False)
+        super(LEDBarGraph, self).__init__(*pins, **kwargs)
         initial_value = kwargs.get('initial_value', 0)
```

Comment:
In that case, should it be documented as `*pins, pwm=False, initial_value=0` ? (with the obvious caveat that `pwm` and `initial_value` can only be provided by name, and not by position)